src/status_im2/contexts/wallet/create_account/view.cljs

```diff
@@ -19,17 +20,28 @@
     (i18n/label :t/keypair-title {:name first-name})))
 
 (defn get-keypair-data
-  [name derivation-path]
+  [name derivation-path color]
```

Comment:
this `color` is  `user-color` or `account-color`? I think it would be nice to clarify which one 👍